skiko/src/commonMain/kotlin/org/jetbrains/skiko/Events.kt

```diff
@@ -8,14 +8,36 @@ object SkikoMouseButtons {
     const val MIDDLE = 1 shl 2
 }
 
+enum class SkikoGestureEventKind {
+    PRESS, TAP, PAN, PINCH, ROTATION, LONGPRESS, SWIPE, UNKNOWN
+}
+enum class SkikoGestureEventDirection {
+    UP, DOWN, LEFT, RIGHT, UNKNOWN
+}
+enum class SkikoGestureEventState {
+    PRESS, STARTED, CHANGED, ENDED, UNKNOWN
+}
+expect class SkikoGesturePlatformEvent
+data class SkikoGestureEvent(
+    val x: Double, val y: Double,
+    val velocity: Double = 0.0,
+    val direction: SkikoGestureEventDirection = SkikoGestureEventDirection.UNKNOWN,
+    val interval: Long = 0,
+    val rotation: Double = 0.0,
+    val scale: Double = 1.0,
+    val kind: SkikoGestureEventKind,
+    val state: SkikoGestureEventState = SkikoGestureEventState.UNKNOWN,
+    val platform: SkikoGesturePlatformEvent? = null
+)
+
 expect class SkikoPlatformInputEvent
 data class SkikoInputEvent(
```

Comment:
Do we plan add SkikoKeyboardModifiers?

We can retrieve them via `event.modifiersEx` in jvm:
```
e.modifiersEx and InputEvent.CTRL_DOWN_MASK != 0
```